### PR TITLE
ZCS-14910 : delete old SMIME jar before upgrade

### PR DESCRIPTION
**Pull request by umagmrit**

### CODE REVIEW

**Review by uttamtakalkar — APPROVED** [review]

File: rpmconf/Spec/Scripts/zimbra-store.pre [hunk 1]
```diff
@@ -23,3 +23,9 @@ fi
 if [ -d "/opt/zimbra/mailboxd/work/zimbraAdmin" ]; then
   rm -rf /opt/zimbra/mailboxd/work/zimbraAdmin > /dev/null 2>&1
 fi
+
+if [ -d /opt/zimbra/lib/ext/smime ]; then
```
Comment (RakeshAMore) [inline rpmconf/Spec/Scripts/zimbra-store.pre]: we need to invest more time to investigate why this is with SMIME extension only.

**Review by RakeshAMore — APPROVED** [review]

**Review by jiteshsojitra — APPROVED** [review]

File: rpmconf/Spec/Scripts/zimbra-store.pre (hunk 1 above)
Comment (dasiyogesh) [reply rpmconf/Spec/Scripts/zimbra-store.pre]: @umagmrit during every upgrade we expect the extension to get cleared and add new ones?

Comment (umagmrit) [reply rpmconf/Spec/Scripts/zimbra-store.pre]: yes @dasiyogesh

**Review by dasiyogesh — APPROVED** [review]

**Review by suchitgupta11 — APPROVED** [review]

**Review by GaneshAnarse — APPROVED** [review]

File: rpmconf/Spec/Scripts/zimbra-core.pre [hunk 2]
```diff
@@ -36,3 +36,9 @@ else
     echo "zimbra hard nofile 524288" >> /etc/security/limits.conf
     chmod 640 /etc/security/limits.conf
 fi
+
+if [ -d /opt/zimbra/lib/ext/com_zimbra_ssdb_ephemeral_store ]; then
```
Comment (jiteshsojitra) [inline rpmconf/Spec/Scripts/zimbra-core.pre]: Please use double quote everywhere.

`if [ -d "/opt/zimbra/lib/ext/com_zimbra_ssdb_ephemeral_store" ]; then`

Comment (umagmrit) [reply rpmconf/Spec/Scripts/zimbra-core.pre]: updated same

**Review by dasiyogesh — APPROVED** [review]

**Pull request merged**